### PR TITLE
releases: Create 1.8/beta bundle

### PR DESCRIPTION
**Pull request by orfeas-k**
Introduces `1.8/beta` bundle

### CODE REVIEW

File: releases/1.8/beta/README.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+
+For any Kubernetes, follow the [installation instructions][install].
+
+## Testing
+
+To deploy this bundle and run tests locally, do the following:
+
+1. Set up Kubernetes, Juju, and deploy the bundle you're interested in (`kubeflow` or
+   `kubeflow-lite`) using the [installation guide](https://charmed-kubeflow.io/docs/install/). This
```
Comment (DnPlas) [inline releases/1.8/beta/README.md:27]: We don't have kubeflow-lite anymore, you can remove any reference to that.

File: releases/1.8/beta/README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+   - lite bundle (using environment variables):
+      ```
+      export KUBEFLOW_AUTH_USERNAME=user1234@email.com
+      export KUBEFLOW_AUTH_PASSWORD=user1234
+      tox -e tests -- -m lite
+      ```
+
+Subsets of the tests are also available using pytest's substring expression selector (e.g.:
+`tox -e tests -- -m full --username user123@email.com --password user123 -k 'selenium'` to run just
+the selenium tests).
```
Comment (DnPlas) [inline releases/1.8/beta/README.md:59]: I'd ditch all these instructions and just point to the [uats](https://github.com/canonical/charmed-kubeflow-uats) and/or the [bundle testing](https://github.com/canonical/bundle-kubeflow/tree/main/tests-bundle#tests-bundle)

File: releases/1.8/beta/README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+## Testing
+
+To deploy this bundle and run tests locally, do the following:
+
+1. Set up Kubernetes, Juju, and deploy the bundle you're interested in (`kubeflow` or
+   `kubeflow-lite`) using the [installation guide](https://charmed-kubeflow.io/docs/install/). This
+   must include populating the `.kube/config` file with your Kubernetes cluster as the active
+   context. Do not create a namespace with the same name as the username, this will cause 
+   pipelines tests to fail. Beware of using `admin` as the dex-auth static-username as the tests 
+   attempt to create a profile and `admin` conflicts with an existing default profile.
```
Comment (DnPlas) [inline releases/1.8/beta/README.md:31]:
```suggestion
```

File: releases/1.8/beta/bundle.yaml [hunk 4]
```diff
@@ -0,0 +1,253 @@
...
+      istio.gateway.namespace: kubeflow
+      istio.gateway.name: kubeflow-gateway
+    _github_repo_name: knative-operators
+  kserve-controller:
+    charm: kserve-controller
+    channel: latest/beta
+    scale: 1
+    trust: true
+    options:
+      deployment-mode: serverless
```
Comment (DnPlas) [inline releases/1.8/beta/bundle.yaml:152]: kserve-controller deploys with serverless mode by default, no need to specify here.

**Review by DnPlas — COMMENTED** [review]
Thanks @orfeas-k, some minor things.

File: releases/1.8/beta/bundle.yaml (hunk 4 above)
Comment (orfeas-k) [reply releases/1.8/beta/bundle.yaml:152]: good catch, removing it. Note that we have this in latest/beta and latest/edge too.

**Review by DnPlas — APPROVED** [review]

**Pull request merged**